applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -281,44 +266,101 @@ choice BT_AUDIO_BROADCAST_BAP_CONFIGURATION
 	bool "Recommended Broadcast codec capability"
 	depends on TRANSPORT_BIS
 	help
 	  Recommended broadcast settings for the nRF5340_audio application
-	  48kHz, 96kbps, 4 retransmits, 20ms transport latency, and 10ms presentation delay
+	  48kHz, 96kbps, 4 retransmits, 20ms transport latency, and 10ms presentation delay.
 
 config BT_AUDIO_BROADCAST_16_2_1
 	bool "16_2_1"
 	depends on TRANSPORT_BIS
 	help
 	  Broadcast mandatory codec capability 16_2_1
-	  16kHz, 32kbps, 2 retransmits, 10ms transport latency, and 40ms presentation delay
+	  16kHz, 32kbps, 2 retransmits, 10ms transport latency, and 40ms presentation delay.
 
 config BT_AUDIO_BROADCAST_24_2_1
 	bool "24_2_1"
 	depends on TRANSPORT_BIS
 	help
 	  Broadcast codec capability 24_2_1
-	  24kHz, 48kbps, 2 retransmits, 10ms transport latency, and 40ms presentation delay
+	  24kHz, 48kbps, 2 retransmits, 10ms transport latency, and 40ms presentation delay.
 
 config BT_AUDIO_BROADCAST_16_2_2
 	bool "16_2_2"
 	depends on TRANSPORT_BIS
 	help
 	  Broadcast mandatory codec capability 16_2_2
-	  16kHz, 32kbps, 4 retransmits, 60ms transport latency, and 40ms presentation delay
+	  16kHz, 32kbps, 4 retransmits, 60ms transport latency, and 40ms presentation delay.
 
 config BT_AUDIO_BROADCAST_24_2_2
 	bool "24_2_2"
 	depends on TRANSPORT_BIS
 	help
 	  Broadcast codec capability 24_2_2
-	  24kHz, 48kbps, 4 retransmits, 60ms transport latency, and 40ms presentation delay
+	  24kHz, 48kbps, 4 retransmits, 60ms transport latency, and 40ms presentation delay.
 endchoice
 
+config BT_AUDIO_BROADCAST_NAME
+	string "Broadcast name"
+	depends on TRANSPORT_BIS
+	default "NRF5340_BROADCASTER"
+	help
+	  Name of the broadcast; not the same as BT_DEVICE_NAME.
+
+config BT_AUDIO_BROADCAST_NAME_ALT
+	string "Alternative broadcast name"
+	depends on TRANSPORT_BIS
+	default "NRF5340_BROADCASTER_ALT"
+	help
+	  Alternative name of the broadcast.
+
+config BT_USE_AUDIO_BROADCAST_NAME_ALT
+	bool "Use the alternative broadcast name"
+	default n
+	depends on TRANSPORT_BIS
+	help
+	  Use the alternative broadcast name.
+
+config BT_AUDIO_BROADCAST_ENCRYPTED
+	bool "Encrypted broadcast"
+	depends on TRANSPORT_BIS
+	default n
+	help
+	  Encrypt the broadcast to limit the connectability
+	  (NOTE: Encrypting broadcast source is not yet supported by host.)
+
+config BT_AUDIO_BROADCAST_ENCRYPTION_KEY
+	string "Broadcast encryption key"
+	depends on TRANSPORT_BIS
+	default "NRF5340_BIS_DEMO"
+	help
+	  Key to use for encryption and decryption; max 16 characters.
+
+config BT_AUDIO_USE_BROADCAST_ID_RANDOM
+	bool "Use a random broadcast ID"
+	depends on TRANSPORT_BIS
+	default y
+	help
+	  Use a randomly generated broadcast ID.
+
+config BT_AUDIO_BROADCAST_ID_FIXED
+	hex "Fixed broadcast ID"
+	depends on TRANSPORT_BIS
+	default 0x123456
+	help
+	  Fixed broadcast ID; 3 octets. Will only be used if BT_AUDIO_USE_BROADCAST_ID_RANDOM=n.
+	  Should only be used for debugging.
+
+config AURACAST
```

Comment:
Or should we enable this as default :P ?

Reply:
I guess we could